decidim-bulletin_board-ruby/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.7.1] - 2021-01-26
```

Comment:
Don't add the version yet. This will be done in the `main` branch before releasing it.